connect-inject/container_init.go

```diff
@@ -169,6 +169,13 @@ services {
     alias_service = "{{ .ServiceName }}"
   }
 }
+
+services {
+  id   = "${POD_NAME}-{{ .ServiceName }}"
```

Comment:
nit: would it be nicer to manage if the service name were first? I'm assuming pod name is long and random looking so it would possibly be cleaner for humans to scan if the readable service name was first?

Reply:
These will all get grouped together under the service `name` already. At the service level though, it's more important to be able to identify the specific instance first, rather than the service. I'm going to leave this for now, but I'm open to changing it later based on community feedback.